### PR TITLE
Expand example for path_prefix and staging environment

### PR DESCRIPTION
**Pull request by sauy7**
Adds a rackup configuration file to illustrate how to support a path_prefix and modifies sinatra app to use it with the staging environment

### CODE REVIEW

File: examples/sinatra/config.ru [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require File.expand_path('fishbrain_example', File.dirname(__FILE__))
+
+use OmniAuth::Builder do
+  configure do |config|
```
Comment (dalen) [inline examples/sinatra/config.ru:4]: can't this be in the `use OmniAuth::Builder do` in the sinatra app itself?

File: examples/sinatra/views/index.haml [hunk 2]
```diff
@@ -7,8 +7,8 @@
     %h1
       Welcome
 
-    %a{ href: "/auth/fishbrain" }
+    %a{ href: "/client/auth/fishbrain" }
       Log in with Fishbrain
 
-    %a{ href: "/auth/cognito-idp" }
+    %a{ href: "/client/auth/cognito-idp" }
```
Comment (dalen) [inline examples/sinatra/views/index.haml]: this link is a leftover from some old test and should actually be removed :)

File: examples/sinatra/config.ru (hunk 1 above)
Comment (sauy7) [reply examples/sinatra/config.ru:4]: It can, I've added this to the README.md

File: examples/sinatra/views/index.haml (hunk 2 above)
Comment (sauy7) [reply examples/sinatra/views/index.haml]: Removed

**Review by dalen — APPROVED** [review]

**Pull request merged**